commit aa7b615c72961aa3cae61518c9112c14ac1e0a47
Author: Christoph Hellwig <hch@lst.de>
Date:   Thu Aug 17 12:24:47 2017 +0200

    blk-mq-pci: add a fallback when pci_irq_get_affinity returns NULL
    
    commit c005390374957baacbc38eef96ea360559510aa7 upstream.
    
    While pci_irq_get_affinity should never fail for SMP kernel that
    implement the affinity mapping, it will always return NULL in the
    UP case, so provide a fallback mapping of all queues to CPU 0 in
    that case.
    
    Signed-off-by: Christoph Hellwig <hch@lst.de>
    Reviewed-by: Omar Sandoval <osandov@fb.com>
    Signed-off-by: Jens Axboe <axboe@kernel.dk>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1647_aa7b615c72961aa3cae61518c9112c14ac1e0a47.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1647_aa7b615c72961aa3cae61518c9112c14ac1e0a47.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
